testsuite/integration/manualmode/pom.xml

```diff
@@ -383,8 +397,62 @@
                     </execution>
+                    <execution>
+                        <id>multi-channel-provisioning</id>
+                        <goals>
+                            <goal>provision</goal>
+                        </goals>
+                        <phase>${multi.channel.provisioning.phase}</phase>
+                        <configuration>
+                            <provisioning-dir>${project.build.directory}/wildfly-multi-channel</provisioning-dir>
+                            <record-provisioning-state>false</record-provisioning-state>
+                            <log-provisioning-time>${galleon.log.time}</log-provisioning-time>
+                            <offline-provisioning>false</offline-provisioning>
+                            <overwrite-provisioned-server>true</overwrite-provisioned-server>
+                            <galleon-options>
+                                <jboss-maven-dist>false</jboss-maven-dist>
+                                <jboss-fork-embedded>${galleon.fork.embedded}</jboss-fork-embedded>
+                                <!-- TODO needing to add this seems not user-friendly -->
+                                <ignore-not-excluded-layers>true</ignore-not-excluded-layers>
+                            </galleon-options>
+                            <channels>
+                                <channel>
+                                    <manifest>
+                                        <groupId>${ee.maven.groupId}</groupId>
+                                        <artifactId>wildfly-ee-channel</artifactId>
+                                        <version>${ee.maven.version}</version>
+                                    </manifest>
+                                </channel>
+                                <channel>
+                                    <manifest>
+                                        <groupId>org.wildfly.prospero</groupId>
+                                        <artifactId>prospero-wildfly-galleon-pack</artifactId>
+                                        <version>${version.org.wildfly.prospero}</version>
+                                    </manifest>
+                                </channel>
+                                <channel>
+                                    <manifest>
+                                        <url>${project.baseUri}/target/test-classes/product-conf-override-manifest.yaml</url>
+                                    </manifest>
+                                </channel>
+                            </channels>
+                            <feature-packs>
+                                <feature-pack>
+                                    <groupId>${ee.maven.groupId}</groupId>
+                                    <artifactId>wildfly-ee-galleon-pack</artifactId>
+                                    <version>${ee.maven.version}</version>
+                                </feature-pack>
+                                <feature-pack>
```

Comment:
Is it necessary to include prospero feature pack? IIUC the test only verifies the interaction between wildfly channel and the product-conf-override-manifest.yaml. The integration tests in prospero already verify [installing Wildfly + prospero feature pack](https://github.com/wildfly-extras/prospero/blob/main/integration-tests/src/test/java/org/wildfly/prospero/it/featurepacks/WildflyFpTest.java).

Reply:
Right; @spyrkob -- that can be removed. That's left over from when I also used this execution for the testing I later moved into https://github.com/wildfly/wildfly-core/pull/5724.

Reply:
Fixed.